gfdlvitals/util/xrtools.py

```diff
@@ -86,6 +86,9 @@ def xr_weighted_avg(dset, weights):
             if sorted(dset[x].dims) == sorted(weight.dims):
                 _dset[x] = dset[x]
 
+        if isinstance(weight, xr.DataArray):
+            weight = weight.fillna(0.0)
+                  �
```

Comment:
There is an unusual character on line 91.

Reply:
I noticed that in my testing as well. It should now be removed.